plugins/filter/dict_kv.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2020 Stanislav German-Evtushenko (@giner) <ginermail@gmail.com>
+#
+# This file is part of Ansible
```

Comment:
Can you use the short GPL header here (i.e. https://github.com/ansible-collections/community.general/blob/main/plugins/filter/time.py#L3)? I think we're trying to use the short one for new stuff (and adjust existing headers by time), so having the short one in a new filter is definitely better :)